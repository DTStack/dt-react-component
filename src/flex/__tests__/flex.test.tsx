import React from 'react';
import { cleanup, render } from '@testing-library/react';

import Flex from '..';

const children = (
    <>
        <div>123</div>
        <div>123</div>
        <div>123</div>
        <div>123</div>
    </>
);

describe('Test Flex Button', () => {
    beforeEach(() => cleanup());

    it('Match the snapshot', () => {
        expect(render(<Flex>{children}</Flex>).asFragment()).toMatchSnapshot();

        expect(render(<Flex vertical>{children}</Flex>).asFragment()).toMatchSnapshot(
            'primary button'
        );
    });
});
