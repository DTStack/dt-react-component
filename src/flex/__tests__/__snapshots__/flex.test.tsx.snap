// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Flex Button Match the snapshot 1`] = `
<DocumentFragment>
  <section
    class="dtc-flex"
    style="flex-wrap: nowrap; justify-content: normal; align-items: normal; gap: 0;"
  >
    <div>
      123
    </div>
    <div>
      123
    </div>
    <div>
      123
    </div>
    <div>
      123
    </div>
  </section>
</DocumentFragment>
`;

exports[`Test Flex Button Match the snapshot: primary button 1`] = `
<DocumentFragment>
  <section
    class="dtc-flex dtc-flex__vertical"
    style="flex-wrap: nowrap; justify-content: normal; align-items: normal; gap: 0;"
  >
    <div>
      123
    </div>
    <div>
      123
    </div>
    <div>
      123
    </div>
    <div>
      123
    </div>
  </section>
</DocumentFragment>
`;
